.github/workflows/daily-nightly-jobs.yml

```
@@ -395,7 +359,7 @@ jobs:
     strategy:
       matrix:
         ceph-image-tag:
-          ["latest-main-devel", "latest-quincy-devel", "latest-pacific-devel"]
+          ["latest-main-devel", "latest-quincy-devel", "latest-reef-devel"]
```

The CI title says `upgrade-from-quincy-stable-to-quincy-devel`. I guess this wasn't right before also?

```suggestion
          ["latest-main-devel", "latest-quincy-devel", "latest-quincy-devel"]
```

---

This is under the `rgw-multisite-test-with-ceph-devel`, right? So the test was missing reef tests previously.